.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: scala
 scala:
-  - 2.11.8
+  - 2.12.0
```

Comment:
Hmm, now I'm confused. I've merged in my 2.12 PR, so this line should no longer be in the diff.

Sorry, do you mind doing another rebase against master? I think GitHub is a bit messed up.